deploy/role.yaml

```diff
@@ -25,6 +25,7 @@ rules:
   - get
   - update
   - patch
+  - list
```

Comment:
Are create/delete/patch/update needed anymore?

Reply:
edited the roles, create delete get are needed